core/src/main/java/jenkins/management/ShutdownLink.java

```diff
@@ -28,38 +28,61 @@
 import hudson.model.ManagementLink;
 import hudson.security.Permission;
 import jenkins.model.Jenkins;
+import net.sf.json.JSONObject;
 import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+import org.kohsuke.stapler.verb.POST;
+
+import javax.servlet.ServletException;
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 import edu.umd.cs.findbugs.annotations.NonNull;
 
-/**
- * @author <a href="mailto:nicolas.deloof@gmail.com">Nicolas De Loof</a>
- */
-@Extension(ordinal = Integer.MIN_VALUE) @Symbol("shutDown")
+@Extension(ordinal = Integer.MIN_VALUE)
+@Symbol("prepareShutdown")
```

Comment:
Incompatible symbol change. Unsure whether we care but should not go without comment.

If we're changing it, should use the term "quiet down" IMO.

Reply:
not even sure why we need a symbol for a management link?

Reply:
🤷 Ask KK.

But might be as simple as having a generic `ExtensionFilter`, configurable within JCasC.

Reply:
- [ ] change symbol to "prepareQuietdown"